.github/workflows/publish-release.yml

```diff
@@ -69,7 +73,7 @@ jobs:
           devCentralToken: ${{ secrets.BALLERINA_DEV_CENTRAL_ACCESS_TOKEN }}
           ballerinaBotWorkflow: $ {{ secrets.BALLERINA_BOT_WORKFLOW }}
         run: |
-          ./gradlew build -Pversion=${VERSION}
+          ./gradlew build -Pversion=${VERSION} -x centralTests -x test
```

Comment:
```suggestion
          ./gradlew build -Pversion=${VERSION}
```

I hope these were skipped for testing work

Reply:
Yes they were skipped for the purpose of testing I shall include them.